src/plugins/shared_ux/public/i18n/index.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+export * from './labs';
```

Comment:
copy-pasta error?

Reply:
Removed in [e27b041](https://github.com/elastic/kibana/pull/124372/commits/e27b041db0704bc27f8125401d58831b3e720cd9) thanks!